asset-transfer-basic/chaincode-java/README.md

```diff
@@ -0,0 +1,14 @@
+# Java FabCar contract sample
```

Comment:
This readme can be removed. We can document the samples after other things are in place.